src/App.tsx

```diff
@@ -1,14 +1,26 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [isLoad, setIsLoad] = useState(true);
+  const [isModalSee, setIsModalSee] = useState(false);
```

Comment:
The state variable `isModalSee` is not named according to common conventions. Consider renaming it to `isModalVisible` for better readability and to follow the convention of using present participles for boolean state variables.